spec/features/admin/admin_invoice_show_spec.rb

```diff
@@ -39,7 +39,7 @@
 
     expect(current_path).to eq(admin_invoice_path(@invoice_1))
     expect(page).to have_content(@invoice_1.id)
-    expect(page).to have_content(@invoice_1.status)
+    expect(page).to have_content(@invoice_1.status.to_s.capitalize)
```

Comment:
With the enum, do we need to turn it back into a string?

Reply:
I took it out the .to_s